receiver/tlscheckreceiver/README.md

```diff
@@ -0,0 +1,27 @@
+# TLS Check Receiver
+
```

Comment:
```suggestion

<!-- status autogenerated section -->
<!-- end autogenerated section -->

```
Adding this will allow `make generate` to generate the header that all components have in their README 👍

Reply:
Thank you. I was wondering why none of the autogenerated code did not appear when I ran `make generate`